developer/system/gui.md

```diff
@@ -90,9 +90,11 @@ Clipboard sharing implementation
 Certainly, it would be insecure to allow AppVM to read/write the clipboards of other AppVMs unconditionally.
 Therefore, the following mechanism is used:
 
-- there is a "qubes clipboard" in dom0 - its contents are stored in a regular file in dom0.
+- there is a "qubes clipboard" in dom0 - its contents are stored in a regular file in dom0 as `/run/qubes/qubes-clipboard.bin`.
 - if the user wants to copy local AppVM clipboard to qubes clipboard, she must focus on any window belonging to this AppVM, and press **Ctrl-Shift-C**. This combination is trapped by `qubes-guid`, and `CLIPBOARD_REQ` message is sent to AppVM. `qubes-gui` responds with `CLIPBOARD_DATA` message followed by clipboard contents.
+- by default, `qubes-guid` limits the maximum clipboard size to 64000 bytes. This could be changed by passing `--max-clipboard-size=SIZE` option to `qubes-guid`. Accepted values are between 256 to 256000 bytes.
```

Comment:
Info about config option? (after all that's how it gets set in practice)

Reply:
Ok. I will delete it.